app/assets/javascripts/jquery/plugins/inat/generic_autocomplete.js

```diff
@@ -65,6 +65,9 @@ $.fn.genericAutocomplete = function ( acOptions ) {
   var options = acOptions || { };
   var ac;
   var field = this;
+  if ( $('html[dir="rtl"]').length > 0 ){
+    options.position = { my: "right top", at: "right bottom" };
```

Comment:
Glad this worked. However, this means that the `position` can never be customized in RTL. Instead, something like this would probably be safer:

```javascript
options.position = options.position || { my: "right top", at: "right bottom" };
```